### PR TITLE
Bump CoreDNS version and update manifest

### PR DESCRIPTION
**Pull request by rajansandeep**

### CODE REVIEW

File: deploy/addons/coredns/coreDNS-clusterrole.yaml [hunk 1]
```diff
@@ -16,3 +16,9 @@ rules:
   verbs:
   - list
   - watch
+- apiGroups:
```
Comment (tstromberg) [inline deploy/addons/coredns/coreDNS-clusterrole.yaml]: Where did this part of the change come from? Is it a required part of the 1.2.4 update?

Comment (rajansandeep) [reply deploy/addons/coredns/coreDNS-clusterrole.yaml]: @tstromberg This allows the federation plugin to fetch zone and region labels.
more details here: https://github.com/coredns/deployment/pull/100

**Comment by RA489** [comment]
@rajansandeep please re-run the test. rest LGTM

**Comment by tstromberg** [comment]
@minikube-bot OK to test

**Comment by rajansandeep** [comment]
No longer applicable as CoreDNS and kube-dns will not be maintained by the add-on manager.
/close

**Pull request closed without merging**